tests/Application/.env

```diff
@@ -12,7 +12,7 @@ APP_SECRET=EDITME
 # Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # For a sqlite database, use: "sqlite:///%kernel.project_dir%/var/data_%kernel.environment%.db.db"
 # Set "serverVersion" to your server version to avoid edge-case exceptions and extra database calls
-DATABASE_URL=mysql://root@127.0.0.1/sylius_%kernel.environment%?serverVersion=5.5
+DATABASE_URL=mysql://root:12344@127.0.0.1/sylius_%kernel.environment%?serverVersion=5.5
```

Comment:
I'm not sure why this change is done? If it's a local configuration, you should use .env.local file.